lib/actions/config.coffee

```diff
@@ -231,17 +231,17 @@ exports.generate =
 
 		Examples:
 
-			$ balena config generate --device 7cf02a6
-			$ balena config generate --device 7cf02a6 --generate-device-api-key
-			$ balena config generate --device 7cf02a6 --device-api-key <existingDeviceKey>
-			$ balena config generate --device 7cf02a6 --output config.json
-			$ balena config generate --app MyApp
-			$ balena config generate --app MyApp --output config.json
-			$ balena config generate --app MyApp \
+			$ balena config generate --device 7cf02a6 --version 2.12.7
+			$ balena config generate --device 7cf02a6 --version 2.12.7 --generate-device-api-key
+			$ balena config generate --device 7cf02a6 --version 2.12.7 --device-api-key <existingDeviceKey>
+			$ balena config generate --device 7cf02a6 --version 2.12.7 --output config.json
+			$ balena config generate --app MyApp --version 2.12.7
+			$ balena config generate --app MyApp --version 2.12.7 --output config.json
+			$ balena config generate --app MyApp --version 2.12.7 \
 				--network wifi --wifiSsid mySsid --wifiKey abcdefgh --appUpdatePollInterval 1
 	'''
 	options: [
-		commandOptions.optionalOsVersion
+		commandOptions.osVersion
```

Comment:
I don't understand -- why do you make it a required argument again?

Reply:
Ignore me, this is another command.

Reply:
Sorry I didn't include a description in the commit :disappointed: 
Just for tracking reasons: This is currently required by the SDK's `os.os.getConfig()` which throws `An OS version is required when calling os.getConfig` when it isn't provided. `balena os configure` tries to find the versions from the image provided, but `balena configure` doesn't work on a specific image.